source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py

```diff
@@ -464,6 +464,15 @@ def _config_resolution(self, launcher_args: dict):
             if distributed_train:
                 self.device_id = self.local_rank
                 launcher_args["multi_gpu"] = False
+            # limit CPU threads to minimize thread context switching
```

Comment:
NIT: Can we have slightly more explanation on why this is good to do? Just for future us to know it as well :)